### ISSUE TITLE
should rstudio/r-base include libgit2?

### ISSUE THREAD

**Issue by kevinushey** [rstudio/r-docker#105]
From https://stackoverflow.com/questions/76801837/docker-cannot-build-my-r-package-fails-to-install-r-dependencies.

Do we have a philosophy around whether the `r-base` image should be considered a "batteries-included" image, so that the more commonly-used R packages will function, or is it intended to be more minimal?

**Comment by glin** [comment]
Yeah, these images are intentionally minimal with just R + Pandoc + TinyTeX, and so far we've just pointed users to Rocker for more batteries-included images in the readme. Reading the SO post, I don't think these images have ever had libgit2 pre-installed, so that's odd. We also haven't changed the dependencies installed here in a long time.
